pyccl/halos/profiles/profile_base.py

```diff
@@ -47,35 +47,25 @@ def __init__(self):
                             "_real or _fourier implementation.")
         self.precision_fftlog = FFTLogParams()
 
-    @property
-    @abstractmethod
-    def normprof(self) -> bool:
-        """Normalize the profile in auto- and cross-correlations by
-        :math:`I^0_1(k\\rightarrow 0, a|u)`
-        (see :meth:`~pyccl.halos.halo_model.HMCalculator.I_0_1`).
-        """
-
-    # TODO: CCLv3 - Rename & allocate _normprof_bool to the subclasses.
+    def normalization(self, hmc, cosmo, a):
```

Comment:
1. Can we rename this to `get_normalization` if it's going to be a public function? `normalization` would imply it is a property.
2. Since we depart from CCL's standard arg ordering, can we make all args kw-only?
3. Are we sure this is never going to need any additional args (i.e. the only things it needs are `cosmo` and `a`?

Reply:
1. Renamed.
2. I've gone with cosmo, a, hmc, which makes more sense.
3. Yes. Think of this as an even more specific version of `fourier`, which does not admit any other arguments.